tests/cyclers/exclusions/reference.log

```diff
@@ -0,0 +1,103 @@
+2016-07-04T09:37:25Z INFO - [baz.20000102T0000Z] -(current:submitted)> started at 2016-07-04T09:37:24Z
+2016-07-04T09:37:25Z INFO - [baz.20000102T0000Z] -(current:running)> succeeded at 2016-07-04T09:37:24Z
+2016-07-04T09:37:25Z INFO - [bar.20000102T0000Z] -(current:submitted)> started at 2016-07-04T09:37:24Z
+2016-07-04T09:37:25Z INFO - [bar.20000102T0000Z] -(current:running)> succeeded at 2016-07-04T09:37:24Z
+2016-07-04T09:37:25Z INFO - [nip.20000102T0000Z] -('job-logs-register', 1) will run after P0Y (after 2016-07-04T09:37:25Z)
+2016-07-04T09:37:25Z INFO - [qux.20000102T0000Z] -('job-logs-register', 1) will run after P0Y (after 2016-07-04T09:37:25Z)
+2016-07-04T09:37:25Z INFO - [baz.20000102T0000Z] -('job-logs-register', 1) will run after P0Y (after 2016-07-04T09:37:25Z)
+2016-07-04T09:37:25Z INFO - [bar.20000102T0000Z] -('job-logs-register', 1) will run after P0Y (after 2016-07-04T09:37:25Z)
+2016-07-04T09:37:26Z INFO - [nip.20000102T0600Z] -triggered off []
+2016-07-04T09:37:26Z INFO - [qux.20000102T1200Z] -triggered off []
+2016-07-04T09:37:27Z INFO - [nip.20000102T0600Z] -submit_method_id=18393
+2016-07-04T09:37:27Z INFO - [nip.20000102T0600Z] -submission succeeded
+2016-07-04T09:37:27Z INFO - [qux.20000102T1200Z] -submit_method_id=18394
+2016-07-04T09:37:27Z INFO - [qux.20000102T1200Z] -submission succeeded
+2016-07-04T09:37:27Z INFO - [qux.20000103T0000Z] -holding (beyond suite stop point) 20000102T1200Z
+2016-07-04T09:37:27Z INFO - [qux.20000103T0000Z] -waiting => held
+2016-07-04T09:37:27Z INFO - [nip.20000102T0600Z] -(current:submitted)> started at 2016-07-04T09:37:26Z
+2016-07-04T09:37:27Z INFO - [qux.20000102T1200Z] -(current:submitted)> started at 2016-07-04T09:37:26Z
+2016-07-04T09:37:28Z INFO - [nip.20000102T1200Z] -triggered off []
+2016-07-04T09:37:28Z INFO - [nip.20000102T0600Z] -(current:running)> succeeded at 2016-07-04T09:37:27Z
+2016-07-04T09:37:28Z INFO - [qux.20000102T1200Z] -(current:running)> succeeded at 2016-07-04T09:37:27Z
+2016-07-04T09:37:28Z INFO - [nip.20000102T0600Z] -('job-logs-register', 1) will run after P0Y (after 2016-07-04T09:37:28Z)
+2016-07-04T09:37:28Z INFO - [qux.20000102T1200Z] -('job-logs-register', 1) will run after P0Y (after 2016-07-04T09:37:28Z)
+2016-07-04T09:37:28Z INFO - [client-command] get_cylc_version osanders@eld668.desktop.frd.metoffice.com:cylc-gui 3385400d-7efc-449b-a6ac-a8698a23101d
+2016-07-04T09:37:29Z INFO - [nip.20000102T1200Z] -submit_method_id=18613
+2016-07-04T09:37:29Z INFO - [nip.20000102T1200Z] -submission succeeded
+2016-07-04T09:37:29Z INFO - [nip.20000102T1200Z] -(current:submitted)> started at 2016-07-04T09:37:28Z
+2016-07-04T09:37:29Z INFO - [nip.20000102T1200Z] -(current:running)> succeeded at 2016-07-04T09:37:29Z
+2016-07-04T09:37:29Z INFO - [nip.20000102T1200Z] -('job-logs-register', 1) will run after P0Y (after 2016-07-04T09:37:29Z)
+2016-07-04T09:37:30Z INFO - Suite shutting down at 2016-07-04T09:37:30Z
```

Comment:
Note we're now removing the cruft from reference logs: https://github.com/cylc/cylc/pull/1892#issuecomment-226602511  (not automated yet, unfortunately).